OWL2/scripts/runManchester.hs

```diff
@@ -24,7 +24,7 @@ import Text.ParserCombinators.Parsec
 processFile :: String -> IO ()
 processFile file = do
   str <- readFile file
-  case runParser (basicSpec << eof) () file str of
+  case runParser (parseOntologyDocument << eof) () file str of
```

Comment:
I get an error here, the PrefixMap is missing.

Reply:
Fixed